src/utils/rest-api.js

```diff
@@ -225,12 +225,17 @@ export function getExportUrl(studyUuid, exportFormat) {
 export function renameElement(studyUuid, newStudyName) {
     console.info('Renaming study ' + studyUuid);
     const renameElementUrl =
-        PREFIX_DIRECTORY_SERVER_QUERIES +
-        `/v1/directories/${studyUuid}/rename/${newStudyName}`;
-
+        PREFIX_DIRECTORY_SERVER_QUERIES + `/v1/elements/${studyUuid}`;
```

Comment:
Maybe modify variables names from `study...` to `element...` since element is not necessary a study. And change console.log too.

Reply:
Done